src/index.d.ts

```diff
@@ -441,6 +441,8 @@ export interface ParserOptions<T = any> extends RequiredOptions {
   locStart: (node: T) => number;
   locEnd: (node: T) => number;
   originalText: string;
+  astFormat: string;
+  printer: Printer<T>;
```

Comment:
I don't think they should be used by anyone. 
Can you share how they used in plugin?

Reply:
We use it to get the [comment printer function](https://github.com/prettier-solidity/prettier-plugin-solidity/blob/924efb60a17076d568e87cc78ff04b8680b1ebed/src/common/printer-helpers.js#L20) but I guess we can just import that directly.
Not really needed now that I look at it.
Up to you if you wanna merge or not.

Reply:
Suggest import it directly.